### PR TITLE
Add Facebox auth

### PR DESCRIPTION
**Pull request by robmarkcole**
**Description:**
Adds Facebox auth and removes teach events

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant/pull/15439) (if applicable):** home-assistant/home-assistant#15439

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/image_processing.facebox.markdown [hunk 1]
```diff
@@ -109,25 +118,6 @@ A valid service data example:
 ```
 {% endraw %}
 
-An `image_processing.teach_classifier` event is fired for each service call, providing feedback on whether teaching has been successful or unsuccessful. In the unsuccessful case, the `message` field of the `event_data` will contain information on the cause of failure, and a warning is also published in the logs. An automation can be used to receive alerts on teaching, for example, the following automation will send a notification with the teaching image and a message describing the status of the teaching:
```
Comment (arsaboo) [inline source/_components/image_processing.facebox.markdown:112]: Is this service being removed?

Comment (robmarkcole) [reply source/_components/image_processing.facebox.markdown:112]: Yes, in the latest PR

**Review by arsaboo — DISMISSED** [review]
Looks good..can be merged when the parent PR is merged.

File: source/_components/image_processing.facebox.markdown (hunk 1 above)
Comment (MartinHjelmare) [reply source/_components/image_processing.facebox.markdown:112]: The service isn't removed but the event fired on service call.

Comment (robmarkcole) [reply source/_components/image_processing.facebox.markdown:112]: Sorry yes thats correct

**Comment by robmarkcole** [comment]
@arsaboo parent PR has been merged now

File: source/_components/image_processing.facebox.markdown [hunk 2]
```diff
@@ -24,6 +24,7 @@ MB_KEY="INSERT-YOUR-KEY-HERE"
 
 sudo docker run --name=facebox --restart=always -p 8080:8080 -e "MB_KEY=$MB_KEY"  machinebox/facebox
 ```
+If you are on an unsecured network you can run Facebox with a username and password, add ` -e "MB_BASICAUTH_USER=my_username" -e "MB_BASICAUTH_PASS=my_password" `
```
Comment (MartinHjelmare) [inline source/_components/image_processing.facebox.markdown]: The requests are still not encrypted right? So we shouldn't recommend this on an unsecured network?

Comment (robmarkcole) [reply source/_components/image_processing.facebox.markdown]: I hadn't considered that, but you are correct.

Comment (robmarkcole) [reply source/_components/image_processing.facebox.markdown]: I guess the statement should be: 'You can run Facebox .. but bare in mind that the component does not encrypt these credentials'?

File: source/_components/image_processing.facebox.markdown [hunk 3]
```diff
@@ -24,6 +24,7 @@ MB_KEY="INSERT-YOUR-KEY-HERE"
 
 sudo docker run --name=facebox --restart=always -p 8080:8080 -e "MB_KEY=$MB_KEY"  machinebox/facebox
 ```
+You can run Facebox with a username and password by adding ` -e "MB_BASICAUTH_USER=my_username" -e "MB_BASICAUTH_PASS=my_password" ` but bare in mind that the component does not encrypt these credentials and this approach does not guarantee security on an unsecured network.
```
Comment (MartinHjelmare) [inline source/_components/image_processing.facebox.markdown]: The formatting of the literal text within back-tics looks weird in the rendered page. Check the netlify preview.

**Comment by robmarkcole** [comment]
ping @MartinHjelmare

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**